src/test/java/teammates/test/cases/browsertests/AdminHomePageUiTest.java

```diff
@@ -220,7 +220,7 @@ private void testCreateInstructorAction() throws Exception {
         coursesPage.loadInstructorHomeTab();
         instructorHomePage = AppPage.getNewPageInstance(browser, InstructorHomePage.class);
         instructorHomePage.clickFeedbackSessionViewResultsLink(demoCourseId, "Second team feedback session")
-                          .waitForPageToLoad();
+                          .expandPanels();
```

Comment:
cc @wkurniawan07 what do you think about these tests? Now that we no longer auto load, I think we can just skip expanding of panels to save on operations and test time, the side-effect is that we will have a lot of HTML changes in this PR and I don't mind that, I would like your opinion